README.md

```diff
@@ -72,7 +72,7 @@ val configuration = RealmConfiguration(schema = setOf(Person::class, Dog::class)
 ```
 
 ```Kotlin
-val realm = Realm.open(configuration)
```

Comment:
I don't have a strong opinion against it, but one benefit of `Realm.open` is to control the caching strategy (since Realms are frozen we can choose internally to implement a singleton by `RealmConfiguration` for example, doing so via the constructor might feel a bit weird. Also `open` strongly indicates opening a resource that you need to handle and close

Reply:
The idea is that we don't really need caching with the frozen architecture...You just create the instance once and use it everywhere.

Reply:
I don't have any strong opinions either, but I see Nabil's point.